experiments/20161104-pol-assay-high-throughput/20161117-second-run.md

```diff
@@ -17,6 +17,30 @@ For one replicate:
 6. Add 32 ul of appropriate transfection mixture to each well.
 
 
-2016-11-17 Notes:
+**2016-11-17 Notes:**
 - 2.7 million cells/mL -> added 3mL of DMEM to the 1mL solution, seeded 4ul of diluted solution for 30000 cells/well.
 - **Dilution:** Seeded three wells each of 1:1, 1:2, 1:4, and 1:8 dilutions. 1:1 contained 30000 cells.
+
+**2016-11-19 Transfection Notes:**
+- Uneven growth of cells in wells
+- The first sample of Vic 1:1 dilution does not appear to contain any cells.
```

Comment:
Make sure that you note later on whether this is a reason for excluding a sample - and yes, it can be a reason for doing so.